packages/reshowcase/reshowcase.5.3.0/opam

```diff
@@ -0,0 +1,49 @@
+opam-version: "2.0"
+synopsis: "A tool to create demos for your ReasonReact components"
+maintainer: ["Denis Strelkov <denis.strelkov@ahrefs.com>"]
+authors: ["bloodyowl <mlbli@me.com>"]
+license: "MIT"
+tags: ["melange" "org:ahrefs"]
+homepage: "https://github.com/ahrefs/reshowcase"
+doc: "https://github.com/ahrefs/reshowcase"
+bug-reports: "https://github.com/ahrefs/reshowcase/issues"
+depends: [
+  "dune" {>= "3.8"}
+  "ocaml"
+  "melange" {>= "2.0.0"}
+  "reason" {>= "3.10.0"}
+  "reason-react"
+  "reason-react-ppx"
+  "ocaml-lsp-server" {with-test}
```

Comment:
Maybe for this one `with-dev-setup` is intended instead?
```suggestion
  "ocaml-lsp-server" {with-dev-setup}
```

Reply:
sounds good (looking forward to 2.2 😄 )

Reply:
(I figured out that this package likely not requiring `ocaml-lsp-server` to build / test and installing it when not necessary is actually a little wasteful)